src/main/java/tracko/commons/util/DateTimeUtil.java

```diff
@@ -0,0 +1,14 @@
+package tracko.commons.util;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A class for formatting LocalDateTime.
+ */
+public class DateTimeUtil {
+    private static final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy <HH:mm>");
```

Comment:
since this is `final` field, perhaps the variable name should be in upper snake case?

Reply:
Good catch, I'll change it! Thanks